AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStore/src/Structure.dfy

```diff
@@ -425,6 +493,50 @@ module {:options "/functionSyntax:4" } Structure {
     ]
   }
 
+  function ReplaceMutableContext(
+    item: map<string, string>,
```

Comment:
Sort of?
It is not a DynamoDB item,
or it would be `map<string, DDB.Types.AttributeValue>`,
and it would not include TABLE_FIELD,
which is the logical key store name.

Ryan would call this item the Encryption Context.
That is entirely correct.
(Unless the cipher-text gets represented as a string, which I know it does not.)
As long as the reader knows that the custom encryption context is a subset of the encryption context.

I will rename this parameter original encryption context.

Reply:
No, sorry, I think BranchKeyContext is clearly the correct name.